crates/net/network/src/peers/manager.rs

```diff
@@ -876,6 +938,8 @@ pub(crate) enum PeerCommand {
     ReputationChange(PeerId, ReputationChangeKind),
     /// Get information about a peer
     GetPeer(PeerId, oneshot::Sender<Option<Peer>>),
+    /// Get reputation and address information on all peers
+    GetPeers(oneshot::Sender<Vec<(i32, NodeRecord)>>),
```

Comment:
I'd like this to return `Vec<Peer>` instead, can be included in the `PeersInfo` trait.

Reply:
I need some way to retrieve the `PeerId` for each one. As it isn't in the `Peer`, I'd need this to return a `Vec<(PeerId, Peer)>`

Reply:
I see, then let's do `Vec<NodeRecord>`